.circleci/config.yml

```diff
@@ -12,20 +12,17 @@ workflows:
       - codacy/sbt:
           name: compile_test
           cmd: |
-            sbt "set scalafmtUseIvy in ThisBuild := false;
+            sbt -mem 2048 "set scalafmtUseIvy in ThisBuild := false;
                  scalafmt::test;
                  test:scalafmt::test;
-                 sbt:scalafmt::test"
-            sbt ++2.11.12 test
-            sbt ++2.12.10 test
-            sbt ++2.13.1 test
-            sbt "coverage;
-                 coverageReport;
-                 coverageAggregate"
+                 sbt:scalafmt::test;
+                 +test;
+                 coverage;
+                 coverageReport"
             if [ -z "$CODACY_PROJECT_TOKEN" ]; then
               echo "CODACY_PROJECT_TOKEN not found. Skipping send coverage to Codacy."
             else
```

Comment:
No need for the `if` statement. The coverage reporter does this check

Reply:
Right!

Reply:
Done ✅